docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -159,8 +159,9 @@ lifetimes are as follows:
 * **Scoped** services are created per scope. In a web application, every web request creates a new unique service scope. That means scoped services are generally created per web request.
 * **Singleton** services are created per DI container. That generally means that they are created only one time per application and then used for whole the application life time.
 
-It is highly recommended that the schema is registered as a singleton. This provides the best performance as
-the schema does not need to be built for every request. As all graph types are constructed at the
+> It is _highly_ recommended that the schema is registered as a singleton. This provides the best performance as
+the schema does not need to be built for every request. This also means that your server is doing way less work and 
+hence reduced CPU utilization. As all graph types are constructed at the
```

Comment:
I think the added sentence is a bit redundant, but fine.  I do like the emphasis placed here.  It would be better if we could give a performance comparison, such as "100 times slower", but this is entirely dependent on the complexity of the schema, and whether they are using the `EnableReadDescriptionFromXmlDocumentation` option and/or expression builders to create their schema.

:+1:

Reply:
Perhaps something like "Having a scoped schema can easily slow down execution by 100x with even a small schema, and much more with a large schema."  Just for reference, benchmarks indicate that running the "hero" query on a very simple schema only takes 1% of the build time, but running an introspection request on the same simple schema takes twice as long to execute as it does to build.  So while build time can be slow, it's certainly possible that execution time can be just as long for a complex query (meaning that even then it will run twice as fast if it didn't need to build).  But that's assuming a small schema with no expression builders.

Reply:
Got it. Yes, I think I ended up being redundant here. I will reword this and push the changes. Thanks for noticing it.